### DISCUSSION TITLE
Does email obfuscation work on link text?

### DISCUSSION THREAD

**Discussion by lvclark** [quarto-dev/quarto-cli#6421] [Q&A] [answered]
### Description

I am using email obfuscation with javascript on my site, and am delighted that you have made this feature available.  Right now I am using it like

```
[email](mailto:janedoe@amazing.net)
```

but I would kind of like it to be

```
[janedoe@amazing.net](mailto:janedoe@amazing.net)
```

however, I am not sure whether the link text would also get obfuscated in the second example.

**Comment by dragonstyle** [comment, marked as answer]
Glad you like it (Pandoc is the one doing the work and it is a very cool feature!). It obfuscates both the text and the link. Here is some example output:

```html
<p><script type="text/javascript">
<!--
h='&#112;&#x6f;&#x73;&#x69;&#116;&#46;&#x63;&#x6f;';a='&#64;';n='&#x63;&#104;&#x61;&#114;&#108;&#x65;&#x73;';e=n+a+h;
document.write('<a h'+'ref'+'="ma'+'ilto'+':'+e+'" clas'+'s="em' + 'ail">'+e+'<\/'+'a'+'>');
// -->
</script><noscript>charles at posit dot co</noscript></p>
```

FWIW, it will also process a simpler version of the link if you'd like `<email@example.com>` and will make the mailto link and obfuscate it for you.

**Reply by lvclark** [reply]
Awesome, thank you for the quick answer!
